sklearn/decomposition/_pca.py

```diff
@@ -53,7 +53,8 @@ def _assess_dimension(spectrum, rank, n_samples):
     Notes
     -----
     This implements the method of `Thomas P. Minka:
-    Automatic Choice of Dimensionality for PCA. NIPS 2000: 598-604`
+    Automatic Choice of Dimensionality for PCA. NIPS 2000: 598-604
```

Comment:
I think that we could remove the note and add the sentence in the long discussion on the top of the docstring and add a proper `References` section.

Reply:
done, but for consistency with how the References section looks in the main PCA doctring, I left the sentence also in the Referneces section.